src/docs/self-hosted/backup.mdx

```diff
@@ -2,54 +2,124 @@
 title: "Self-Hosted Backup & Restore"
 ---
 
-## Quick Backup
+## Partial JSON backup
 
-If you need a quick way to backup and restore your Sentry instance and you don't need historical event data, you can use the included backup and restore scripts in the `self-hosted/scripts` directory. These commands will save and load all project and user data, but will not contain any event data. Run these from your `self-hosted` directory.
-As of 23.3.1, we're enabling users to opt in/out of error reporting for these scripts.
+If you need a quick way to backup, view, examine, and restore data from your Sentry instance and
+don't need historical event data, you can use the included backup and restore scripts in the
+`self-hosted/scripts` directory (on self-hosted versions >=23.3.1), via `sentry-admin.sh`
```

Comment:
I think it would be a nicer for readability if these go from newest -> oldest, thoughts?
1. `sentry-admin.sh` 
2. backup/restore scripts
3. export and import commands

Reply:
Done.